drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1745,6 +1745,7 @@ void TextureStorage::_update_render_target(RenderTarget *rt) {
 		if (status != GL_FRAMEBUFFER_COMPLETE) {
 			glDeleteFramebuffers(1, &rt->fbo);
 			GLES3::Utilities::get_singleton()->texture_free_data(rt->color);
+			GLES3::Utilities::get_singleton()->texture_free_data(rt->depth);
```

Comment:
```suggestion
			if (rt->overridden.color.is_null()) {
				GLES3::Utilities::get_singleton()->texture_free_data(rt->color);
			}
			if (rt->overridden.depth.is_null()) {
				GLES3::Utilities::get_singleton()->texture_free_data(rt->depth);
			}
```
akien-mga's concern is correct and it is a pre-existing problem with this code, we shouldn't try to free the GL texture data when the textures come from an external source

Reply:
Eep, I suspect it was my oversight that led to this pre-existing problem. In any case, the suggested code looks good to me :-)

Reply:
Thank you for suggested code, and it looks good to me.